src/kic/api/serializers.py

```diff
@@ -484,3 +513,47 @@ def save(self, **kwargs):
             raise serializers.ValidationError(
                 {api_settings.NON_FIELD_ERRORS_KEY: sync_error.args[0]}
             ) from sync_error
+
+
+class ProductSerializer(serializers.Serializer):
+    code = serializers.CharField(
+        max_length=20,
+        source="product_code",
+        help_text=get_help_text("datamodel.VerzoekProduct", "product_code"),
+    )
+
+
+class VerzoekProductSerializer(serializers.HyperlinkedModelSerializer):
+    product_identificatie = ProductSerializer(
+        source="*",
+        required=False,
+        allow_null=True,
+        help_text=_(
+            "Identificerende gegevens van het PRODUCT voor het geval er bij `product` geen URL kan worden opgenomen bij naar het PRODUCT in de Producten en Diensten API."
```

Comment:
```suggestion
            "Identificerende gegevens van het PRODUCT voor het geval er bij `product` geen URL kan worden opgenomen naar het PRODUCT in de Producten en Diensten API."
```

Also nice if you split that string over multiple lines to keep it < 88 chars long (black!)